include/swift/AST/NameLookup.h

```diff
@@ -27,21 +27,38 @@
 #include "swift/Basic/SourceLoc.h"
 #include "swift/Basic/SourceManager.h"
 #include "llvm/ADT/SmallVector.h"
+#include "swift/AST/TypeRepr.h"
 
 namespace swift {
 class ASTContext;
+class ASTWalker;
 class DeclName;
 class Type;
 class TypeDecl;
 class ValueDecl;
 struct SelfBounds;
 class NominalTypeDecl;
+class Evaluator;
 
 namespace ast_scope {
 class ASTSourceFileScope;
 class ASTScopeImpl;
 } // namespace ast_scope
 
+using CollectedOpaqueReprs = SmallVector<TypeRepr *, 2>;
```

Comment:
Since NameLookup.h already includes TypeRepr.h, you don't need to restate the type alias here